wizeline/project-lab
# Feature/update Wizeline Branding - #148 - Update logo and Project Lab name

#### What does this PR do?

- Update the Wizeline logo to the new branding image on the top nav, favicon and loader.
- Change Proposal Hunt name for Project Lab

#### Where should the reviewer start?

You should check the result in:
- The tab
- The top nav
- The loader (refresh the page and see the logo)

#### How should this be manually tested?

1. Pull the changes and run the project
2. Check the favicon on the tab
3. Check the logo and the Project name in the top nav
4. Refresh any page and check the logo in the loader

#### What are the relevant tickets?

#148 

#### Screenshots

![image](https://user-images.githubusercontent.com/96010361/150412416-e116dee3-2ff5-4392-908a-90144d4f7bda.png)
